meiki_ui/src/components/app/toolbar/SyncIndicator.svelte

```diff
@@ -11,7 +11,7 @@
 
 {#if show}
     {#if error}
-        <Badge label={`${error.toUpperCase()} !`} type={BadgeType.ERROR}>
+        <Badge label={error.toUpperCase()} type={BadgeType.ERROR}>
```

Comment:
Decided to remove the exclamation because it was more inline with not having the `saved` and `saving` text, it looked less simple with the warning triangle already present